src/components/Commit.vue

```diff
@@ -199,6 +203,7 @@ export default {
 
   methods: {
     showInfo() {
+      storage.delete('commitInfoAccepted');
       this.$store.commit('setAcceptCommitInfo', false);
```

Comment:
Couldn’t saving and deleting this from storage be handled in the store mutations.

Reply:
Believe this has been sorted in the recent [commit](https://github.com/aphtoken/aph-desktop-web-dex/pull/6/commits/125ff4b3bb1d2b9c7d482c2fc9fc96b7508b8909)? .. Also, done for both dex and commit.